jdbc/pom.xml

```diff
@@ -74,8 +74,7 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-common</artifactId>
-      <version>2.7.2</version>
-      <scope>provided</scope>
+      <version>${hadoop.common.version}</version>
     </dependency>
```

Comment:
Sure got it, also, have moved hadoop-* profile to root level pom.xml, does this looks ok ?